sdks/go/pkg/beam/core/runtime/graphx/schema/schema.go

```diff
@@ -28,16 +28,15 @@ package schema
 import (
 	"fmt"
 	"reflect"
-	"strconv"
 	"strings"
-	"sync/atomic"
 
 	"github.com/apache/beam/sdks/go/pkg/beam/core/graph/coder"
 	"github.com/apache/beam/sdks/go/pkg/beam/core/sdf"
 	"github.com/apache/beam/sdks/go/pkg/beam/core/util/reflectx"
 	"github.com/apache/beam/sdks/go/pkg/beam/internal/errors"
 	pipepb "github.com/apache/beam/sdks/go/pkg/beam/model/pipeline_v1"
 	"github.com/golang/protobuf/proto"
+	"github.com/google/uuid"
```

Comment:
You'll need to add this to the gogradle.lock as well, otherwise the jenkins machines won't get at it properly.

Reply:
20:30:53 > Task :sdks:go:goTest FAILED
20:30:55 .gogradle/project_gopath/src/github.com/apache/beam/sdks/go/pkg/beam/core/runtime/graphx/schema/schema.go:39:2: cannot find package "github.com/google/uuid" in any of:
20:30:55 	/home/jenkins/jenkins-slave/workspace/beam_PreCommit_Go_Commit/src/sdks/go/.gogradle/project_gopath/src/github.com/apache/beam/sdks/go/vendor/github.com/google/uuid (vendor tree)
20:30:55 	/home/jenkins/.gradle/go/binary/1.12.7/go/src/github.com/google/uuid (from $GOROOT)
20:30:55 	/home/jenkins/jenkins-slave/workspace/beam_PreCommit_Go_Commit/src/sdks/go/.gogradle/project_gopath/src/github.com/google/uuid (from $GOPATH)

Reply:
Done, looks like it's working.